.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/src/util/diproche/diprocheProgram
```

Comment:
I wouldn't put the diproche-program in the `src` folder if it's not actual source-code of this project.